packages/paste-core/components/toast/src/ToastContainer.tsx

```diff
@@ -3,15 +3,15 @@ import {Stack} from '@twilio-paste/stack';
 import {ToastPortal} from './ToastPortal';
 import {ToastContainerProps, ToastContainerPropTypes} from './types';
 
-const ToastContainer: React.FC<ToastContainerProps> = ({children, ...props}) => {
+const ToastContainer = React.forwardRef<HTMLDivElement, ToastContainerProps>(({children, ...props}, ref) => {
   return (
-    <ToastPortal {...props}>
+    <ToastPortal {...props} ref={ref}>
```

Comment:
I would move this ref on to the Stack to be more precise on what the user expects the ref to point to